app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -556,6 +552,19 @@ class BrowserTabViewModelTest {
         assertEquals(false, loadingViewState().isLoading)
     }
 
+    @Test
+    fun whenBrowsingAndViewModelGetsProgressUpdateLowerThan50ThenViewStateIsUpdatedTo50() {
+        setBrowserShowing(true)
+
+        testee.progressChanged(0)
+        assertEquals(50, loadingViewState().progress)
```

Comment:
maybe reference the constant instead of using a magic number?

Reply:
Yeah I thought about it but then the constant would have to be public and it kinda put me off, but happy to change it though.

Reply:
fair enough, let's keep it like this! 👍